smartdevicelink/sdl_core
# Send full image paths to HMI in AppServiceData

Fixes #2926 

This PR is **ready** for review.

### Risk
This PR makes **no** API changes.

### Testing Plan
1. Subscribe HMI to WEATHER and NAVIGATION service data
2. Send OnAppServiceData(WEATHER) and OnAppServiceData(NAVIGATION) from mobile services with existing images
3. Verify that HMI receives full file paths for all mobile service images

### Summary
Send full image paths to HMI in AppServiceData, rather than provided filenames

### Changelog
##### Bug Fixes
* Send full image paths to HMI in AppServiceData, rather than provided filenames 

### CLA
